usr/share/rear/skel/default/etc/scripts/system-setup

```diff
@@ -128,19 +138,20 @@ for system_setup_script in /etc/scripts/system-setup.d/*.sh ; do
     if rear_debug ; then
         read -p "Press ENTER to run $( basename $system_setup_script ) "
         set -x
-        source $system_setup_script
+        source $system_setup_script || echo -e "\n'source $system_setup_script' exited with $?\n"
         # The only known way how to do 'set +x' after 'set -x' without a '+ set +x' output:
         { set +x ; } 2>/dev/null
         echo
     else
         echo "Running $( basename $system_setup_script )..."
-        source $system_setup_script
+        source $system_setup_script || echo -e "\n'source $system_setup_script' exited with $?\n"
```

Comment:
Maybe better user the `Error()` function?

Reply:
not sure if you can use it outside ReaR (we are in an init script)

Reply:
In skel/default/etc/scripts/system-setup.d/00-functions.sh
we have a simple 'Error' function for system setup scripts
but at this code place we do not yet have it sourced.
The first one that gets sourced is 00-functions.sh
but for that source call there is no 'Error' function.
Perhaps it works when the 'Error' function would be
the very first thing in 00-functions.sh so that bash
would know the 'Error' function just in time
if sourcing 00-functions.sh fails?

Reply:
No - I cannot use the 'Error' function
because it outputs `ERROR: ...`
but (at least currently) I do not want
to shout "ERROR" at the user only because
a `source FILENAME` results non-zero exit code
which could be rather often false alarm because
'source' returns the status of the last command
executed in FILENAME and I guess (at least currently)
we do not (yet) care much about the exit code
of the last command in our setup scripts.